applications/daos.md

```diff
@@ -0,0 +1,135 @@
+If your project doesn't want to use council and Democracy because the governance resources are limited,
+then you need to write a voting system for group decisions. This is a repetitive job and it doesn't add much value.
+I recommend using DAOS and leaving the governance code writing to the DAOS team.
+We will continue to update, it makes sense, and both KICO and LISTEN are high-quality projects in Kusama and have been given slots.
+
+## Team
+### Team members
+* JimYam
+* wetalice
+* Zifan Zhang
+* You Lam
+### Team's experience
+* JimYam has been engaged in blockchain chain development for 5 years and is passionate about blockchain industry innovation. Currently, he is the main developer of LISTEN parallel chain team. Having worked in Tencent for 3 years, he has unique insights into social products. He has been following polkadot ecological development since Substrate1.0.
+* wetalice Graduated from top5 universities in China, good at economics and game theory. Currently he is the lead developer for the KICO team.
+* Zifan Zhang has worked for NetEase and did front-end development for 6 years. Currently, he has his own entrepreneurial team in Singapore.
+* You Lam is test engineer who has been working in the industry for 9 years.
+### Contact
+* **Contact Name:** JimYam
+* **Contact Email:** transxask1@gmail.com
+### Legal Structure
+* **Registered Address:** N/A
+* **Registered Legal Entity:** N/A
+### Team Code Repos
+
+- [https://github.com/daos-org/daos.git](https://github.com/daos-org/daos.git)
+- [https://github.com/DICO-TEAM/dico-chain.git](https://github.com/DICO-TEAM/dico-chain.git)
+- [https://github.com/listenofficial/listen-parachain.git](https://github.com/listenofficial/listen-parachain.git)
+
+### Development Status
+
+- [https://github.com/daos-org/daos.git](https://github.com/daos-org/daos.git)
+
+***
+
+## Development Roadmap
+### Overview
+* **Total Estimated Duration:** 2 months
+* **Full-Time Equivalent (FTE):** 6FTE
+* **Total Costs:** 10k USD
+### Milestone 1 — Implement Create-Dao, Agency, DoAS, Square, and Sudo Modules
+* **Estimated duration:** 1 month
+* **FTE:** 6
+* **Costs:** 6000 USD
+
+|Number   | Deliverable       | Specification                                                                                                                                                               |
+|-----:  |-------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|0a.     | License           | MIT, APACHE2.0                                                                                                                                                              |
+|0b.     | Documentation     | Write Document for each module                                                                                                                                              |
```

Comment:
Documenting each module is great and useful, but I'd suggest to also provide documentation about how all the modules interact with another.

Reply:
> 记录每个模块非常有用，但我建议还提供有关所有模块如何与另一个模块交互的文档。

In the second milestone, 0a will provide documentation for interacting with other modules